src/components/include/connection_handler/connection_handler.h

```diff
@@ -325,6 +325,14 @@ class ConnectionHandler {
    */
   virtual void CreateWebEngineDevice() = 0;
 
+  /**
+   * @brief Checks if such an application is in any of the connections.
+   * @param connection_key Application's connection key.
+   * @return True if the application present in the any connections.
```

Comment:
@ydementieiev `if the application present in the any connections.` -> `if the application is present in the connections list`

Reply:
@AByzhynar changed, thank you